packages/clay-charts/src/Chart.js

```diff
@@ -11,7 +12,8 @@ import templates from './Chart.soy.js';
 class Chart extends Component {}
 
 Object.assign(Chart.prototype, ChartBase);
-Chart.STATE = ChartBase.STATE;
+Object.assign(Chart.prototype, DataComponent);
+Chart.STATE = Object.assign(ChartBase.STATE, DataComponent.STATE);
```

Comment:
Should this rather be `Object.assign({}, ChartBase.STATE, DataComponent.STATE)`? Otherwise, aren't we mutating `ChartBase.STATE`?

Reply:
Yes we are, good catch thanks